core/blockchain.go

```diff
@@ -239,6 +239,7 @@ type BlockChain struct {
 	engine    consensus.Engine
 	validator Validator // Block and state validator interface
 	processor Processor // Block transaction processor interface
+	verifyManager *VerifyManager
```

Comment:
Shall we use interface here, like `validator`, `processor` ?

Reply:
`verifyManager` is not a good name, we need distinguish it from `validator` here.